src/Microsoft.DotNet.Helix/Sdk/GetHelixWorkItems.cs

```diff
@@ -60,14 +60,26 @@ IDictionary CreateWorkItemMetadata(string name)
                 return metadata;
             }
 
+            ITaskItem2 CreateTaskItem(string workItemName, Dictionary<string, string> metadata)
+            {
+                var workItem = (ITaskItem2) new TaskItem(workItemName);
```

Comment:
We can probably skip this hard cast here, and just have:
```suggestion
                ITaskItem2 workItem = TaskItem(workItemName);
```
That avoids the hard cast and turns a runtime error (if the cast will fail because TaskItem isn't ITaskItem2) into a compile time one, which is always better. :-)